language-examples/csharp/grpc/README.md

```diff
@@ -23,16 +23,18 @@ Please see the details here: [Authentication](../../../README.md).
 Following are the steps that need to be done to run this example:
 
 1. Clone the repo (Please see details here: [cloning guidelines](../../../README.md)).
-2. cd to `vectara-demo/csharp/grpc` directory.
-3. Run the `download_google_protos.sh` script on terminal like following:
+2. cd to `vectara-demo/csharp/common` directory and run `dotnet build common.csproj`. This will build the authentication
```

Comment:
Can you please change the `vectara-demo` to `language-examples`

Reply:
Addressed in 4dee05582ebc92ad5876afa58d2409b04fcd5252